packages/shell-api/src/collection.ts

```diff
@@ -782,7 +784,7 @@ export default class Collection extends ShellApiClass {
   @serverVersions([ServerVersions.earliest, '3.2.0'])
   async update(filter: Document, update: Document, options: UpdateOptions & { multi?: boolean } = {}): Promise<UpdateResult | Document> {
     printDeprecationWarning(
-      'Collection.update() is deprecated. Use updateOne, updateMany or bulkWrite.',
+      'Collection.update() is deprecated. Use updateOne, updateMany, findAndModify, or bulkWrite.',
```

Comment:
Fun fact: `findAndModify` is deprecated as a driver function: https://github.com/mongodb/node-mongodb-native/pull/2740#issuecomment-782399075

I think we want to keep `findAndModify` a shell method, but maybe we don't want to recommend using it anymore?

Reply:
Ah, thanks for pointing this out! Yeah, then it makes sense to me to not include it here. I'll ping Jeff in the ticket about that before removing it from the PR

Reply:
Removed in 6c0dce6